src/Symfony/Component/Finder/Comparator/Comparator.php

```diff
@@ -36,7 +36,7 @@ public function getTarget()
      *
      * @param string $target The target value
      */
-    public function setTarget($target)
```

Comment:
Can you remove the "obsolete" phpdoc (and do the same elsewhere where the phpdoc does not add any interesting information)?

Reply:
Done :-)